citi-onboarding/Delta-odonto
# Feature/navbar

## What i did

- Creation of the navbar

## How to teste

- On one terminal you should type the  following commands to access the client folder and run the Keystone example
`cd client`
`npm start` or `yarn start` 

- Now you should see the navbar with the logo in the left corner and the sections in the navbar "Quem somos - Nossos serviços - Nossos clientes - Blog  - Contato"

- when you set the responsive mode, you should see the menu hamburger int ther rght corner  and the logo in the left corner

- when you click in the menu hamburger you should see the side drawer with the sections Quem somos - Nossos serviços - Nossos clientes - Blog  - Contato column aligned

- when you scroll the page, you should see the navbar change your background color, logo color and text color.
